test/libsyncengine/jobs/network/testnetworkjobs.cpp

```diff
@@ -647,28 +648,28 @@ void TestNetworkJobs::testUploadAborted() {
     CPPUNIT_ASSERT(newNodeId.empty());
 }
 
-void TestNetworkJobs::testUploadSessionConstructorException() {
-    const RemoteTemporaryDirectory remoteTmpDir(_driveDbId, _remoteDirId, "testUploadSessionConstructorException");
+void TestNetworkJobs::testDriveUploadSessionConstructorException() {
+    const RemoteTemporaryDirectory remoteTmpDir(_driveDbId, _remoteDirId, "testDriveUploadSessionConstructorException");
 
     SyncPath localFilePath =
-        localTestDirPath;  // The constructor of UploadSession will attempt to retrieve the file size of directory.
+        localTestDirPath;  // The constructor of DriveUploadSession will attempt to retrieve the file size of directory.
 
     CPPUNIT_ASSERT_THROW_MESSAGE(
-        "UploadSession() didn't throw as expected",
-        UploadSession(_driveDbId, nullptr, localFilePath, localFilePath.filename().native(), remoteTmpDir.id(), 12345, false, 1),
+        "DriveUploadSession() didn't throw as expected",
+        DriveUploadSession(_driveDbId, nullptr, localFilePath, localFilePath.filename().native(), remoteTmpDir.id(), 12345, false, 1),
         std::runtime_error);
 }
 
-void TestNetworkJobs::testUploadSessionSynchronous() {
-    LOGW_DEBUG(Log::instance()->getLogger(), L"$$$$$ testUploadSessionSynchronous");
+void TestNetworkJobs::testDriveUploadSessionSynchronous() {
+    LOGW_DEBUG(Log::instance()->getLogger(), L"$$$$$ testDriveUploadSessionSynchronous");
 
-    const RemoteTemporaryDirectory remoteTmpDir(_driveDbId, _remoteDirId, "testUploadSessionSynchronous");
+    const RemoteTemporaryDirectory remoteTmpDir(_driveDbId, _remoteDirId, "testDriveUploadSessionSynchronous");
 
     SyncPath localFilePath = localTestDirPath / bigFileDirName / bigFileName;
 
-    UploadSession uploadSessionJob(_driveDbId, nullptr, localFilePath, localFilePath.filename().native(), remoteTmpDir.id(),
+    DriveUploadSession DriveUploadSessionJob(_driveDbId, nullptr, localFilePath, localFilePath.filename().native(), remoteTmpDir.id(),
```

Comment:
Do not start variable name with an upper case

Reply:
[fd7ca22](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/fd7ca228ae9ffe8df0c0838a1736f018f3e454af)